spring-kafka-test/src/main/java/org/springframework/kafka/test/context/EmbeddedKafka.java

```diff
@@ -172,5 +173,12 @@
 	 */
 	int zkSessionTimeout() default EmbeddedKafkaBroker.DEFAULT_ZK_SESSION_TIMEOUT;
 
+	/**
+	 * Timeout in seconds for admin operations (e.g. topic creation, close).
+	 * @return default {@link EmbeddedKafkaBroker#DEFAULT_ADMIN_TIMEOUT}
+	 * @since 3.0
```

Comment:
@breader124 The 3.0 release is a long way off; we can back-port this to 2.4.4 if you need this before the end of the year.